README.md

```diff
@@ -67,483 +67,6 @@ We recommend you read the "Architecture" section in our
 [White Paper](https://github.com/artipie/white-paper) to fully
 understand how Artipie is designed.
 
-### Binary Repo
```

Comment:
@Sammers21 let's keep a reference link to these example readmes. I know it will be changed by @yegor256 later, but we should not lose this reference for now.